node/node.go

```diff
@@ -256,19 +255,28 @@ func (node *Node) addPendingStakingTransactions(newStakingTxs staking.StakingTra
 func (node *Node) AddPendingStakingTransaction(
 	newStakingTx *staking.StakingTransaction,
 ) error {
+	if !node.isPreStakingEpoch() {
+		utils.Logger().Info().Str("Hash", newStakingTx.Hash().Hex()).
```

Comment:
There already is a .hex log thing I think

Reply:
This is newly added log. In previous logic, a node not on beacon chain / not in staking epoch received staking transaction will silently return a nil error. A logging only happens when the node is broadcasting the txn. So is it true that this info doesn't have enough value to be logged?

Reply:
No you aren’t. You don’t need to make these kinds of assumptions in a garbage collected language and it’s not necessary to think this deeply over this. Focus on business logic please

Reply:
Removed